pkg/collector/corechecks/oracle/custom_queries_test.go

```diff
@@ -57,6 +57,15 @@ func TestCustomQueries(t *testing.T) {
 	sender.AssertMetricTaggedWith(t, "Gauge", "oracle.custom_query.test.c1", []string{"c2:A"})
 }
 
+func TestCanConnectTags(t *testing.T) {
```

Comment:
Let's move this into a more appropriate file (it's currently with custom query tests)